jobserver/models/reports.py

```diff
@@ -77,6 +77,12 @@ def __str__(self):
     def get_staff_url(self):
         return reverse("staff:report-detail", kwargs={"pk": self.pk})
 
+    @property
+    def is_draft(self):
```

Comment:
I appreciate this is a term we've probably used elsewhere, but would it make more sense to flip it around and have a property called `is_published` instead?

In keeping with the terminology shown on the site for outputs, we only have released and published reports. Released could be a bit confusing to users (and probably devs too), so just referring to published seems a bit nicer than introducing a third option.